### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by rafabernad**
Initial package commit.

### CODE REVIEW

**Review by joeldenning — APPROVED** [review]
Looks good to me. Feel free to merge and see if the publish works in travis. You may need to specify some env variables in Travis to make it work, as described in the instructions.

I've invited you to the npm organization, and will add you to a team that can @esm-bundle/publish lottie-web after that - let me know when you've accepted the invite.

File: rollup.config.js [hunk 1]
```diff
@@ -1,16 +1,46 @@
+import path from "path";
 import commonjs from "@rollup/plugin-commonjs";
 import { terser } from "rollup-plugin-terser";
+import copy from "rollup-plugin-copy";
+import { sync } from "glob-all";
+
+path.dirname(require.resolve("lottie-web/package.json"));
```
Comment (filoxo) [inline rollup.config.js:7]: Is this line needed?

Comment (joeldenning) [reply rollup.config.js:7]: Good point, this looks like dead code.

Comment (rafabernad) [reply rollup.config.js]: Good caught, my fault. I've removed it

**Comment by rafabernad** [comment]
@joeldenning, I've accepted the invite from npm. And added the variables to travis. Just FYI, master branch is not protected, as I don't have permissions to see the settings options.

**Comment by joeldenning** [comment]
👍  I've made you an admin for this github repo and given you publish access.

**Pull request merged**